scanner/indexer/indexer.go

```diff
@@ -81,14 +81,18 @@ func ecosystems(ctx context.Context) []*ccindexer.Ecosystem {
 	es := []*ccindexer.Ecosystem{
 		alpine.NewEcosystem(ctx),
 		dpkg.NewEcosystem(ctx),
-		gobin.NewEcosystem(ctx),
-		java.NewEcosystem(ctx),
-		nodejs.NewEcosystem(ctx),
-		python.NewEcosystem(ctx),
 		rhcc.NewEcosystem(ctx),
 		rhel.NewEcosystem(ctx),
 		rpm.NewEcosystem(ctx),
-		ruby.NewEcosystem(ctx),
+	}
+	if !features.ScannerV4SkipLanguage.Enabled() {
```

Comment:
This PR disables by default the previously enabled feature. Should it be noted somehow, or why it is disabled by default?

Reply:
It currently disables a feature which never existed before. I will invert the flag to make it enable something instead of disabling something. Hopefully this'll make it more clear